pkg/util/metadata/metadata.go

```diff
@@ -233,6 +233,23 @@ func GetDevicePath(volumeID string) string {
 				return diskPaths[0]
 			}
 
+			adressParts := strings.Split(device.Address, ":")
```

Comment:
can we add some comment here so that we can understand why we made changes here?

Reply:
I added a comment with ref to the udev code that creates the soft link.